testsuite/ada_lsp/SA22-032.folding/test.json

```diff
@@ -0,0 +1,173 @@
+[
+   {
+      "comment": [
+         "test automatically generated"
```

Comment:
this needs to be adjusted to say what the test does

Reply:
Done